api/v1alpha1/obcluster_types.go

```diff
@@ -60,8 +60,10 @@ type OBClusterStatus struct {
 //+kubebuilder:object:root=true
 //+kubebuilder:subresource:status
 //+kubebuilder:printcolumn:name="Status",type="string",JSONPath=".status.status"
-//+kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp"
 //+kubebuilder:printcolumn:name="ClusterName",type="string",JSONPath=".spec.clusterName"
+//+kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp"
+//+kubebuilder:printcolumn:name="Image",type="string",JSONPath=".spec.observer.image",priority=1
+//+kubebuilder:printcolumn:name="Storage",type="string",JSONPath=".spec.observer.storage",priority=1
```

Comment:
The `storage` seems not to be in type `string`, what will `kubectl get xxx -o wide` print?

Reply:
![image](https://github.com/user-attachments/assets/9b1855df-6608-4c9b-adbf-fc9c2fc2b8f5)
it's soemthing like this